stellarphot/settings/settings_files.py

```diff
@@ -365,13 +389,32 @@ def save(self, settings):
                     raise ValueError(
                         "Cannot save partial settings when full settings already exist."
                     )
+                # Are we updating or replacing partial settings?
+                if update and self._partial_settings is not None:
+                    # Grab a dict of the settings, only keeping values that are not
+                    # None. Making it dict so that the update method can be used to
+                    # merge the two sets of settings.
+                    temp_settings_arg = {
```

Comment:
I know I am being a stickler, but I don't like the names of these variables because it is not clear which are the settings currently in the partial settings in the object and which are the updates.  Might I suggest instead of `temp_settings_arg` you called it `passed_partial_settings` and instead of `temp_settings_file` you call it `existing_partial_settings` or some variant like that.  Have settings from two sources both starting with `temp` makes the code hard to read.

Reply:
Good suggestion!